OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -385,10 +394,27 @@ - (void)session:(OTSession *)session streamDestroyed:(OTStream *)stream {
     }
 }
 
+- (void)session:(OTSession*) session
+connectionCreated:(OTConnection*) connection {
+    _internalConnectionCount++;
+    
+    //check creationtime of the connections
+    [self compareConnectionTimeWithConnection: connection];
+}
+
+- (void)session:(OTSession*) session
+connectionDestroyed:(OTConnection*) connection {
+    _internalConnectionCount--;
+    
+    //check creationtime of the connections
+    [self compareConnectionTimeWithConnection: connection];
+}
```

Comment:
That's right. Ignore my comment. If this data is provided from the sdk level, that would be better.